evm-contracts/src/v0.6/dev/AggregatorProxy.sol

```diff
@@ -125,7 +137,24 @@ contract AggregatorProxy is AggregatorInterface, AggregatorV3Interface, Owned {
       uint256 answeredInRound
     )
   {
-    return aggregator.getRoundData(_roundId);
+    uint16 reqEpoch;
```

Comment:
We actually lose gas here, because of `uint16` to `unit256` conversion when using the `reqEpoch`, and we do not get any benefits of packing with other storage elements, as this is the only element smaller than 32 bytes.

Consider using `unit256` type.

Reply:
I'd argue that uint16 is cleaner here, because it reflects the semantics of epoch.